RESOURCES/INTHEWILD.md

```diff
@@ -142,6 +142,7 @@ Join our growing community!
 - [Symmetrics](https://www.symmetrics.fyi)
 
 ### Others
+ - [Dropbox](https://www.dropbox.com/)
```

Comment:
```suggestion
 - [Dropbox](https://www.dropbox.com/) [@bkyryliuk]
```

Reply:
```suggestion
 - [Dropbox](https://github.com/dropbox)
```

Reply:
I think the first one " - [Dropbox](https://www.dropbox.com/) [@bkyryliuk]" makes most sense.

Reply:
+1